recipe/build.sh

```diff
@@ -15,6 +15,3 @@ export NETCDF_ROOT=$PREFIX
 make -j$CPU_COUNT
 make check
 make install
-
-# We can remove this when we start using the new conda-build.
-find $PREFIX -name '*.la' -delete
```

Comment:
I assume this step is no longer needed and the "new conda-build" has been around for awhile now?